.github/workflows/deploy.yml

```diff
@@ -110,7 +112,9 @@ jobs:
         with:
           scenario: smoke-test
           docker-image: ${{ needs.docker.outputs.docker-image }}
-          azure-credentials: ${{ secrets.AZURE_CREDENTIALS }}
+          azure-client-id: ${{ secrets.AZURE_CLIENT_ID }}
+          azure-tenant-id: ${{ secrets.AZURE_TENANT_ID }}
+          azure-subscription-id: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
```

Comment:
.github/actions/test-performance doesn't exist? unless I'm missing something.
Which would mean this step can be removed.